receiver_test.go

```diff
@@ -128,3 +130,82 @@ func TestReceiver_Nack_channel_ack_returns_error_it_should_return_the_same_error
 
 	assert.Same(t, expectedErr, err)
 }
+
+func TestReceiver_Receive(t *testing.T) {
+	dd := make(chan amqp.Delivery, 1)
+
+	now := time.Now()
+
+	dd <- amqp.Delivery{
+		Headers: amqp.Table{
+			"X-Custom-Header": []string{"test value1", "test value2"},
```

Comment:
how does it work with a slice of strings? maybe you could also verify here that a single string value in a header works ok

Reply:
It is a good point. I assumed here that all received messages will come from the sender in the library which always publishes arrays. Will add a fix soon.

Reply:
It is a good point. I assumed here that all received messages will come from the sender in the library which always publishes arrays. Will add a fix soon.